src/components/tag/AposTag.vue

```diff
@@ -1,38 +1,46 @@
 <template>
   <div :class="`apos-tag ${modifier}`" :disabled="disabled">
-    <slot name="icon"></slot>
+    <Close class="apos-tag__close-icon" title="Remove Tag" :size="10" />
     {{ label }}
   </div>
 </template>
 
 <script>
-module.exports = {
+import Close from 'vue-material-design-icons/Close.vue';
+export default {
   props: {
     label: String,
     modifier: {
       default: '',
       type: String
-    },
-    disabled: Boolean
+    }
+  },
+  components: {
+    Close
   }
 }
 </script>
 
 <style lang="scss">
 .apos-tag {
   display: inline-flex;
-  padding: 6px 10px;
+  padding: 6px 12px;
+  border-radius: var(--a-border-radius);
   background-color: var(--a-base-1);
   font-size: map-get($font-sizes, default);
   letter-spacing: 0.5px;
-  border-radius: var(--a-border-radius);
   color: var(--a-base-8);
   align-items: center;
   justify-content: center;
+}
 
-  .material-design-icon {
-    display: flex;
-    margin-right: 5px;
+.apos-tag__close-icon {
+  display: flex;
+  position: relative;
+  top: 1px;
```

Comment:
This seems to work with _either_ `display: flex` _or_ the relative `top: 1px` business. With both it's looking off center. I'd go with flex or `inline-flex`, probably.

![Screen Shot 2020-03-24 at 9 48 02 AM](https://user-images.githubusercontent.com/1155984/77439026-8e782200-6db4-11ea-802e-58404718054a.png)

Reply:
Nevermind. You're doing relative/1px on notifications and that one doesn't respond to `flex`, so let's keep it consistent using the relative positioning.